### DISCUSSION TITLE
Cool project. Actively maintained?

### DISCUSSION THREAD

**Discussion by rcontreras-te** [riok/Kreya#256] [General]
Hi all,

Just came across you project.  I thought it was pretty cool.... but I'm wondering how active the project is being maintained?  I see there have only been about 5 commits in the past year.

**Comment by rcontreras-te** [comment]
oh wait... nevermind. my mistake.  not open source. doh!

**Comment by CommonGuy** [comment]
Hi and thanks for your feedback. Just as a heads up, we are currently actively working on the 1.16.0 release :)
